src/lib.rs

```diff
@@ -162,47 +148,23 @@ mod util;
 ///     `param1: Type1, param2: Type2 ...` instead of the field type itself. The parameters are
 ///     transformed into the field type using the expression `expr`. The transformation is performed
 ///     when the setter is called.
-#[proc_macro_derive(TypedBuilder, attributes(builder))]
-pub fn derive_typed_builder(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
-    let input = parse_macro_input!(input as DeriveInput);
-    match impl_my_derive(&input) {
-        Ok(output) => output.into(),
-        Err(error) => error.to_compile_error().into(),
-    }
+pub use typed_builder_macro::TypedBuilder;
+
+#[doc(hidden)]
```

Comment:
Now that it's an official trait in its own crate, I think it should be exposed and documented. I can add the documentation myself later though.

Reply:
To be honest, I don't see a value in exposing the trait. Adding it to the documentation whould be a distraction since it is only used internally in the code generated by the macro.

But adding a normal comment would be nice.